shiptendance/views/index.ejs

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= title %></title>
+    <link rel='stylesheet' href='/stylesheets/style.css' />
+  </head>
+  <body>
+    <h1><%= title %></h1>
```

Comment:
To clean up your views and keep them dry, you could pull this top portion out and create a partial next time.